tools/deploy_metrics_and_monitoring_resources.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+. $(dirname $0)/../multi-platform-support.sh
+DIR=`pwd`
+k=false
+
+usage() { echo "Usage: $0 [-c <cluster>] -n <namespace>"
```

Comment:
Do you actually accept the options without the `=`? Or should it be `-c=<cluster>` etc.?